src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.cs

```diff
@@ -30,8 +31,58 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
         /// </summary>
         internal class TemporaryStorageService : ITemporaryStorageService2
         {
+            /// <summary>
+            /// The maximum size of a single storage unit in a memory mapped file which is shared with other storage
```

Comment:
Could a comment be provided explaining where these numbers came from?  Why 128k in this case for example?  Thanks!